doc/design/quic-programming-guide.md

```diff
@@ -126,11 +126,12 @@ Please follow [Conference Server build instructions](https://github.com/open-web
 Steps to run Conference Server with pre-built binary:
 
 1. Go to the untarred conference server dir, and run `./bin/init-all.sh --deps`; this would try to install mongodb and rabbitmq-server. Don't set any password for rabbitmq or mongodb.
-2. Still in the same dir, run `bin/init-all.sh`.
-3. Update quic_agent/agent.toml, set `hostname` to the hostname quic agent is running on.
-4. run `bin/start-all.sh`.
-5. Open [https://localhost:3004/quic.html](https://localhost:3004/quic.html) on the same host with Chrome browser to visit the web sample page. Due to the test certificate, you may need confirm this unsafe access.
-6. Press 'Start Sending' button to start transferring data to conference server. Press 'Stop Sending' button on the web page to stop sending.. If there's no error in the console, that implies server is correctly setup.
+1. Still in the same dir, run `bin/init-all.sh`.
+1. Update quic_agent/agent.toml, set `hostname` to the hostname quic agent is running on.
+1. Update management_api/management_api.toml, set `enableWebTransport` to `true`.
+1. run `bin/start-all.sh`.
+1. Open [https://localhost:3004/quic.html](https://localhost:3004/quic.html) on the same host with Chrome browser to visit the web sample page. Due to the test certificate, you may need confirm this unsafe access.
```

Comment:
I think ou need to mention the chrome flag related to make sure user don't have issue on chrome 89 and below?

Reply:
It's a web client specific requirement, described in JavaScript SDK. https://github.com/open-webrtc-toolkit/owt-client-javascript/blob/master/docs/design/webtransport.md#limitations.